src/index.js

```diff
@@ -132,6 +153,27 @@ class HawkWebpackPlugin {
     return maps;
   }
 
+  /**
+   * Find commits in the project directory
+   * @return {Commits[]}
+   */
+  findCommits() {
+    const options = {
+      repo: this.gitOptions.repo,
+      number: this.gitOptions.number,
+      fields: ["hash", "subject", "authorEmail", "authorDate"],
+    }
+
+    const commits = gitlog(options);
```

Comment:
do we need try-catch here?

Reply:
It is better to leave it, if the user specified the path incorrectly or something else happened, he could get user-friendly log